app/views/shared/authentication/_forem_creator_signup.html.erb

```diff
@@ -92,7 +92,13 @@
         <% else %>
           <div class="crayons-field mt-6">
             <%= f.label :forem_owner_secret, t("views.auth.register.secret.label"), class: "crayons-field__label" %>
-            <%= f.password_field :forem_owner_secret, class: "crayons-textfield", required: ForemInstance.needs_owner_secret?, placeholder: t("views.auth.register.secret.placeholder") %>
+            <div class="relative js-forem_owner_secret-toggle-wrapper">
+              <%= f.password_field :forem_owner_secret, class: "crayons-textfield js-password", required: ForemInstance.needs_owner_secret?, placeholder: t("views.auth.register.secret.placeholder") %>
+              <button type="button" class="crayons-btn crayons-btn--ghost crayons-btn--s js-creator-password-visibility h-100 absolute z-elevate right-0" aria-label="<%= t("views.auth.register.field.password.show") %>" aria-pressed="false">
```

Comment:
small change needed (blocking) - this button is re-using the `aria-label` "Show password", which means for users of screen readers it won't be clear which button is for the password and which is for the secret.

We'll need to add a new key to the i18n files - you can find these at:

- `config/locales/views/auth/en.yml`
- `config/locales/views/auth/fr.yml`

You'll find there's already a section in the YAML file for 'secret', so we can just add another key inside there, e.g. `show: Show new Forem secret`, and then reference it like:

```suggestion
              <button type="button" class="crayons-btn crayons-btn--ghost crayons-btn--s js-creator-password-visibility h-100 absolute z-elevate right-0" aria-label="<%= t("views.auth.register.secret.show") %>" aria-pressed="false">
```